src/extension.api.ts

```diff
@@ -0,0 +1,5 @@
+import { BazelProjectView } from './types';
+
+export interface BazelVscodeExtensionAPI {
+	readonly parseProjectFile: BazelProjectView;
```

Comment:
```suggestion
	readonly parseBazelProjectFile: BazelProjectView;
```
Just to be consistent with the names `BazelProjectView`, `getBazelProjectFile`, and `bazelprojectparser`.

Reply:
it's already in the `bazel` extension under the `BazelVscodeExtensionAPI` interface. feels a little redundant to add `bazel` to another name; especially since the name of the file extension being processed is `projectview`